Account/FxADeviceRegistration.swift

```diff
@@ -54,7 +54,7 @@ open class FxADeviceRegistration: NSObject, NSCoding {
 
     public convenience required init(coder: NSCoder) {
         let id = coder.decodeObject(forKey: "id") as! String
-        let version = coder.decodeObject(forKey: "version") as! Int
+        let version: Int = coder.decodeObject(forKey: "version") as! Int? ?? 0
```

Comment:
Damnit. I can no longer replicate the missing version situation so I can't go back to look at it. Do we have any startup crashes for iOS that are sync related? If so, this _could_ be it.